code/__DEFINES/evacuation.dm

```diff
@@ -0,0 +1,13 @@
+// States for the evacuation controller
+#define EVACUATION_STATE_IDLE 0
+#define EVACUATION_STATE_INITIATED 1 // Evacuation has begun, but it can be cancelled
+#define EVACUATION_STATE_AWAITING 2 // Awaiting players to board the shuttle/pods/etc, can't be cancelled but can be delayed
+#define EVACUATION_STATE_NORETURN 3 // Shuttle/pods/etc have departed, can't be cancelled nor delayed
```

Comment:
Please rename to prevent confusion with shuttle style points of no return.

suggest: `EVACUATION_STATE_EVACUATED`